### ISSUE TITLE
JxMapViewer.zoomToBestFit(fitPoints, 0.4) throws exception

### ISSUE THREAD

**Issue by Heiko-Zelt** [msteiger/jxmapviewer2#123]
Exception in thread "main" java.lang.IllegalArgumentException: GeoBounds is not valid - minLat must be less that maxLat.
	at org.jxmapviewer.viewer.GeoBounds.setRect(GeoBounds.java:66)
	at org.jxmapviewer.viewer.GeoBounds.<init>(GeoBounds.java:52)
	at org.jxmapviewer.JXMapViewer.zoomToBestFit(JXMapViewer.java:702)

fitPoints = HashSet size = 3
[50.07921600341797, 8.242026329040527]
[50.07921600341797, 8.241934776306152]
[50.07921600341797, 8.24198055267334]

maybe because 3 different points have the same latitude?

**Comment by Heiko-Zelt** [comment]
I have got the fitPoints from EXIF data of photos I took. It's possible that sometimes different photo images accidentally have the same latitude. I want zoomToBestFit() to always work. Either I have to write a workaround for myself or I could deliver a bug fix. I like the concept of test driven development. So I would write a test first. Usually I use JUnit 5. I don't see any JUnit tests in this GitHub repository. So I would add dependencies to build.gradle.
```
dependencies {
    implementation group: 'commons-logging', name: 'commons-logging', version: '1.2'
    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.8.1'
    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.8.1'
}
```

**Comment by msteiger** [comment]
Yes, sounds good to me. I will review the pull request.

**Comment by Heiko-Zelt** [comment]
I wrote 3 unit tests. I used JUnit 4 for compatibility with Java 7.

[JXMapViewerTest.java](https://github.com/Heiko-Zelt/jxmapviewer2/blob/master/jxmapviewer2/src/test/java/org/jxmapviewer/JXMapViewerTest.java)

I think the problem is that "GeoBounds" must have an extend in JxMapViewer. minLat can't be equal to maxLat and minLng can't be equal to maxLng.
```
        if (!(minLat < maxLat))
        {
            throw new IllegalArgumentException("GeoBounds is not valid - minLat must be less that maxLat.");
        }
```
I compared the behavior with Leaflet. In Leaflet "LatLngBounds" can have zero extend in both directions. So it's just a point. For example the Leaflet-method "flyToBounds" can handle this. That would be the perfect solution for me. I don't know changing it in JxMapViewer could break compatibility with older projects. Introducing another Class "GeoBounds0" and another method "fitToBounds0" would bloat the code.
